### ISSUE TITLE
Question about the gradient normalization

### ISSUE THREAD

**Issue by superkevingit** [ShiyuLiang/odin-pytorch#4]
https://github.com/ShiyuLiang/odin-pytorch/blob/34e53f5a982811a0d74baba049538d34efc0732d/code/calData.py#L61
Hi, When I read the code about  the gradient normalization part, I have some questions. What's the meaning for 63, 62.1 and 66.7, and where can I learn these settings? Thank you~

**Comment by YukiNagato** [comment]
i think it might be the variance of the images. See transformer in [cal.py](https://github.com/ShiyuLiang/odin-pytorch/blob/34e53f5a982811a0d74baba049538d34efc0732d/code/cal.py#L27-L30)
```python
start = time.time()
#loading data sets

transform = transforms.Compose([
    transforms.ToTensor(),
    transforms.Normalize((125.3/255, 123.0/255, 113.9/255), (63.0/255, 62.1/255.0, 66.7/255.0)),
])

```
